src/main/webapp/i18n/de/instructorCourseDashboard.json

```diff
@@ -10,11 +10,14 @@
             "tutors": "Tutoren",
             "assessments": "Assessments",
             "open": "offen",
+            "openComplaint": "offen Beshwerden",
+            "openRequest": "offen Anfragen",
```

Comment:
horrible translation! Please fix it!

Reply:
Changed. Can you check if it is better now?